__tests__/app.test.js

```diff
@@ -0,0 +1,149 @@
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+const request = require("supertest");
+const app = require("../db/app/app.js");
+const { readEndpointsFile } = require("../utils.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api", () => {
+  describe("GET", () => {
+    test("returns a 200 status code", () => {
+      return request(app).get("/api").expect(200);
+    });
+    test("should return an object with a key of endpoints countaining objects. JSON should match endpoints.json file ", () => {
```

Comment:
See Rose's post from yesterday about this endpoint - it will make it much easier for you moving forward and will remove the need to update this every time you add another endpoint.